.github/workflows/bump-version-and-update-milestone.yml

```diff
@@ -0,0 +1,47 @@
+#############################################################################
+# GitHub Action to bump release version
+#
+#############################################################################
+name: "Bump version and Update Milestone"
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        description: 'New Version'
+        required: true
+        default: 'warning'
+permissions:
+  contents: write
+  pull-requests: write
+jobs:
+  printInputs:
+    runs-on: ubuntu-latest
+    steps:
+      - run: |
+          echo "Latest Version: ${{ github.event.inputs.version }}"
+  bump:
+    name: Bump version
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout the latest code
+        uses: actions/checkout@v2
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
+      - name: Get the version
+        id: get_version
+        run: |
+          VERSION=$(curl -s https://api.github.com/repos/checkstyle/checkstyle/releases/latest \
+                         | jq -r ".tag_name" | sed 's/checkstyle-//') 
```

Comment:
Version should be taken from user input.
```
inputs:
      version:
        description: 'New Version'
        required: true
```

Reply:
Changes Done